google/ggrc-core
# GGRC-1406 Make project behavior as for other snapshotable objects

# Issue description

‘Project’ should behave as other snapshotable objects.
It should be displayed on the audit page when these objects are mapped to a program. Also it should the possibility to map ‘Project’ through Unified mapper in audit page and other object pages.

# Steps to test the changes

1. Create Project and map it on Program
2. Create Audit and map a Program with mapped project
3. Go to the Audit Info
4. The 'Project' should be displayed on scoped objects.
5. Also it also it should be displayed on 'Add scope' if Project not map on Program and it possible to map on Program through mapper.

# Solution description

Add 'Project' on `all` list on `rules.py` and make it snapshotable.

# Sanity checklist




# PR Review checklist
